okta/plugin.spec.yaml

```diff
@@ -132,6 +132,41 @@ connection:
     type: credential_secret_key
     required: true
 actions:
+  get_factors:
+    title: Get an Okta user's factors
+    description: Returns an object containing all of a user's factors for MFA
+    input:
+      user_id:
+        title: Okta User ID
+        description: The user id of the user you want the factors of
```

Comment:
```suggestion
        description: The user ID of the user you want the factors of
```

Reply:
We no longer require this because ID is not an acronym. Most other vendors use id.

Reply:
https://english.stackexchange.com/questions/101248/how-should-the-abbreviation-for-identifier-be-capitalized

Reply:
Ah, thanks for the resource Mike. Looks like I'm wrong, sorry Luke